test/unit/spec/media/track/remotevideotrack.js

```diff
@@ -414,6 +414,64 @@ describe('RemoteVideoTrack', () => {
       });
     });
   });
+
+  describe('Picture-in-Picture', () => {
+    let el;
+    let setRenderHintsSpy;
+    let track;
+    let originalDocumentPictureInPicture;
+    let pipEls;
+
+    const dispatchVisibilityChangeEvent = async () => {
+      document.visibilityState = 'visible';
+      document.dispatchEvent('visibilitychange');
+      await waitForSometime();
+    };
+
+    beforeEach(() => {
+      documentVisibilityMonitor.clear();
+      originalDocumentPictureInPicture = global.documentPictureInPicture;
+      global.documentPictureInPicture = { window: { document: { querySelectorAll: () => pipEls } } };
+      global.document = global.document || new Document();
+      setRenderHintsSpy = sinon.spy();
+      el = document.createElement('video');
+      track = makeTrack({ id: 'foo', sid: 'bar', setRenderHint: setRenderHintsSpy, options: { enableDocumentVisibilityTurnOff: true } });
+      track.attach(el);
+      pipEls = [el];
+    });
+
+    afterEach(() => {
+      global.documentPictureInPicture = originalDocumentPictureInPicture;
+      if (global.document instanceof Document) {
+        delete global.document;
+      }
```

Comment:
I can see why you are assigning the `documentPictureInPicture` to its original value, but it is unclear why you need to delete the `global.document` after each test.

Reply:
I'm following the same pattern used by other test suites in this file.